src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -271,6 +271,22 @@ public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteB
         {
             return true;
         }
+    },
+    /**
+     * An operator that only performs matching against analyzed columns.
+     */
+    ANALYZER_MATCHES(100)
+    {
+        @Override
+        public String toString()
+        {
+            return ": '<term>'";
+        }
+
+        public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteBuffer rightOperand)
+        {
+            throw new UnsupportedOperationException();
```

Comment:
Let's comment this

Reply:
Sure. And my understanding is that this is an acceptable behavior because the only valid use for the `:` operator is on SAI indexes, and those do not use this method. Further, the operator itself does not have enough context to know the correct answer to `isSatisfiedBy`.